### PR TITLE
Several update for pretrain model

### PR DESCRIPTION
**Pull request by kamo-naoyuki**
- I created new repository to manage pretrain models: https://github.com/espnet/espnet_model_zoo
- Move espent2.bin.zenodo_upload to espnet_model_zoo.zenodo_upload
- I added  --download_model option to the template scripts.

```sh
./run.sh --skip_data_prep false --skip_train true --download_model "model_name"
```

### CODE REVIEW

**Comment by kamo-naoyuki** [comment]
I'll merge this PR. Please use this system and follow up.

**Comment by sw005320** [comment]
Sorry for my late response. I'm busy in these days but will review this later.

File: egs2/TEMPLATE/asr1/asr.sh [hunk 1]
```diff
@@ -81,17 +83,18 @@ feats_normalize=global_mvn  # Normalizaton layer type
...
                                     # e.g.
-                                    # decode_asr_model=train.loss.best.pth
-                                    # decode_asr_model=3epoch.pth
-                                    # decode_asr_model=valid.acc.best.pth
-                                    # decode_asr_model=valid.loss.ave.pth
+                                    # inference_asr_model=train.loss.best.pth
+                                    # inference_asr_model=3epoch.pth
+                                    # inference_asr_model=valid.acc.best.pth
+                                    # inference_asr_model=valid.loss.ave.pth
+download_model= # Download a model from Model Zoo and use it for decoding
```
Comment (sw005320) [inline egs2/TEMPLATE/asr1/asr.sh:97]: `for decoding` --> `for inference`

**Comment by sw005320** [comment]
> Sorry for my late response. I'm busy these days but will review this later.

LGTM.

**Pull request merged**